frontends/web/src/routes/exchange/components/buysell.tsx

```diff
@@ -133,6 +154,22 @@ export const BuySell = ({
           </div>
         )}
       </div>
+      {exchangeDealsResponse?.success &&
+        exchangeDealsResponse.exchanges.some(exchange => exchange.exchangeName === 'pocket') &&
```

Comment:
tiny nit: for readability I'd prefer to wrap JSX in parenthesis
```suggestion
        exchangeDealsResponse.exchanges.some(exchange => exchange.exchangeName === 'pocket') && (
```